secureSubmit.php

```diff
@@ -332,6 +332,7 @@ public function hookHeader()
             return;
         }
 
+        $this->context->controller->addJS('https://js.globalpay.com/v1/globalpayments.js');
         $this->context->controller->addCSS($this->_path.'views/css/securesubmit.css');
         $this->context->controller->addJS($this->_path.'views/js/secure.submit-1.0.2.js');
         $this->context->controller->addJS($this->_path.'views/js/securesubmit.js');
```

Comment:
As proof that things are working as intended can we remove these references and re-test?

Reply:
@MSmedal this one is our securesubmit.js internal javascript file, this one is used to have the mapping of payment fields and further success and error handling and also for form submitting.
This file is mandatory so we can not remove this one.

FYI, this one the actual file is loading there:
https://github.com/mayur-vachchewar/heartland-prestashop-1.7-plugin/blob/master/views/js/securesubmit.js